lisa/src/main/java/it/unive/lisa/program/annotations/values/ClassAnnotationValue.java

```diff
@@ -0,0 +1,50 @@
+package it.unive.lisa.program.annotations.values;
+
+/**
+ * A class annotation value.
+ * 
+ * @author <a href="mailto:vincenzo.arceri@unive.it">Vincenzo Arceri</a>
+ */
+public class ClassAnnotationValue extends BasicAnnotationValue {
```

Comment:
This should be `CompilationUnitAnnotationValue` just to match the naming with the program members